frontend/src/components/features/chat/action-suggestions.tsx

```diff
@@ -9,15 +9,43 @@ import { useUserConversation } from "#/hooks/query/use-user-conversation";
 
 interface ActionSuggestionsProps {
   onSuggestionsClick: (value: string) => void;
+  // For testing purposes
+  conversationIdProp?: string;
+  conversationProp?: { selected_repository?: string | null };
```

Comment:
```suggestion
  conversationId?: string;
  selectedConversationRepository?: string | null;
```

Should also change name within the prop